src/clr-angular/progress/progress-bars/progress-bar.spec.ts

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+*/
+
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ClrProgressBarModule } from './progress-bar.module';
+import { ClrProgressBar } from './progress-bar';
+
+@Component({
+  template: `<clr-progress-bar value="20"></clr-progress-bar>`,
+})
+class TestComponent {}
+
+describe('ClrProgressBar component', () => {
+  describe('View', () => {
+    let fixture: ComponentFixture<any>;
+    let clrProgressBar;
+
+    beforeEach(function() {
+      TestBed.configureTestingModule({
+        declarations: [TestComponent],
+        imports: [ClrProgressBarModule],
+      });
+
+      fixture = TestBed.createComponent(TestComponent);
+      fixture.detectChanges();
+
+      clrProgressBar = fixture.debugElement.query(By.directive(ClrProgressBar)).nativeElement;
+    });
+    it('expect to project content', () => {
```

Comment:
There could be more tests.

1) Test the default displayValue, and overriden value
2) Check the ID is set and linked
3) Test all inputs get set properly